onboard/src/api/api.js

```diff
@@ -0,0 +1,70 @@
+import axios from "axios";
+
+export const BASE_API_URL = process.env.REACT_APP_API_URL
+  ? process.env.REACT_APP_API_URL
+  : "http://localhost:8080";
+
+let myAxios = null;
+
+export const getMyAxios = async () => {
+  if (!myAxios) {
+    myAxios = axios.create({
+      baseURL: BASE_API_URL,
+      withCredentials: true,
+    });
+
+    myAxios.interceptors.response.use(
+      // Any status code that lie within the range of 2xx cause this function to trigger
+      (res) => {
+        return res;
+      },
+      // Any status codes that falls outside the range of 2xx cause this function to trigger
+      (err) => {
+        if (err.response.status !== 401) {
```

Comment:
401 = unauthorized. What is the point of explicitly excluding 401 from promise rejection?